Cargo.toml

```diff
@@ -52,3 +53,11 @@ glutin = { version = "0.12", optional = true }
 env_logger = "0.4"
 notify = "4"
 rand = "0.3"
+
+[[example]]
```

Comment:
I'm curious. Does registering only some of the examples here still allows running the others? I.e. have you tried running other examples with this change?

Reply:
Oops. Nope, will fix.

Reply:
ouch, too bad I removed them. Maybe you can just grab that from my changeset to not rewrite by hand